src/Nethermind/Nethermind.Core/Extensions/Bytes.cs

```diff
@@ -531,7 +531,7 @@ public State(byte[] bytes, int leadingZeros, bool withZeroX, bool withEip55Check
         [DebuggerStepThrough]
         public static string ByteArrayToHexViaLookup32Safe(byte[] bytes, bool withZeroX)
         {
-            if (bytes.Length == 0)
```

Comment:
why do we thinkg that an empty string is ok to return from this method? I think the null should never be passed in here in the first place